binder/environment.yml

```diff
@@ -16,6 +16,3 @@ dependencies:
   - matplotlib
   - imageio
   - pip
```

Comment:
Is the pip dependency still needed now that we don't pip install anything?

Reply:
I'd like to leave it, because I've found it useful to be able to install stuff from the notebooks to try things out when working out what to put in notebooks for new releases.

Reply:
Sounds good.